design-proposals/network-binding-plugin/network-binding-plugin.md

```diff
@@ -387,6 +388,11 @@ Alternatives passing the domain attachment to sync VMI:
 Both of the options are not perfect, but the `virt-handler.SyncVMI`
 has fewer cons. Therefore, it was chosen.
 
+#### Additional resource requests for the virt-launcher compute container
+Some binding plugins may require an additional binary to be shipped inside the virt-launcher compute image.
+This binary requires additional resources that have to be explicitly stated.
```

Comment:
I do not think we need to limit it to an additional binary. It could be just more resources from the cgroupv2 of the compute container, consumed by an existing binary (e.g. libvirt).

Reply:
Done.